### ISSUE TITLE
Conscrypt isn't installed if RepoListActivity isn't the entry point

### ISSUE THREAD

**Issue by kaeptmblaubaer1000** [maks/MGit#392]
If I close an app using home button, it directly gets killed by the LMK.
If I close MGit while in `RepoDetailActivity` and then reopen and try to pull, fetch or push it fails with the original SSL error.
This is because the SSL provider is installed in `RepoListActivity`. Since somebody requested to add an automated pull, which requires a `Service` or something else that isn't an `Activity`, we'd get problems while implementing that.
It should be installed after ACRA in the `Application`.

**Comment by maks** [comment]
Yes good point, I can't remember why we would have wanted to install it in the activity and not App singleton.

**Comment by kaeptmblaubaer1000** [comment]
@maks I'm implementing a fix for this, but since there's currently absolutely no `Timber` logging available in `SGitApplication` (even if using the debug mode) it'll fix #376 too to make it available.

**Comment by maks** [comment]
No worries. I replied on #376 issue about debugtree use. Please do make the Debugtree change in a separate PR though.
